File: src java/Inch.java
public class Inch { //Дюймы в см
    public static void main(String[] args) {
        int inchcount = 1;
        double sum = 0;
        double inch = 2.54;
        int cycle = 20;
        while (inchcount <= cycle) {
            sum = inchcount * inch;
            System.out.println(inchcount + " дюйм = " + sum + " см");
            inchcount++;
        }
    }
}

File: src/Inch.kt
//Дюймы в см
class Inch {
    fun inchKotlin() {
        var inchcount = 1
        var sum = 0.0
        val inch = 2.54
        val cycle = 20
        while (inchcount <= cycle) {
            sum = inchcount * inch
            println("$inchcount дюйм = $sum см")
            inchcount++
        }
    }
}